assets/js/post-form.js

```diff
@@ -104,6 +105,48 @@
         })
     }
 
+    PostForm.prototype.initImagePasting = function() {
+        var self = this
+
+        self.$markdownEditor.bind('paste', function (event) {
+            if (event.type.indexOf('copy') === 0 || event.type.indexOf('paste') === 0) {
+                event.clipboardData = event.originalEvent.clipboardData;
```

Comment:
@marzochi Our JS code style is no semicolons currently, so would you mind removing them from this line and any other line that contains them.
```suggestion
                event.clipboardData = event.originalEvent.clipboardData
```